app/controllers/toggle-moto-mask-security-code/maskSecurityCode.get.controller.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const { response, renderErrorView } = require('../../utils/response')
+
+module.exports = async (req, res) => {
+  try {
+    const pageData = {
+      allowMoto: req.account.allow_moto,
+      motoMaskCardNumberInputEnabled: req.account.moto_mask_card_number_input,
+      motoMaskCardSecurityCodeInputEnabled: req.account.moto_mask_card_security_code_input
+    }
+
+    return response(req, res, 'toggle-moto-mask-security-code/index', pageData)
+  } catch (error) {
+    return renderErrorView(req, res, false, error.errorCode)
+  }
```

Comment:
Do we need a 'return' response(req, res, 'toggle-moto-mask-security-code/index', pageData), or is it implicit

Reply:
You're correct no need for return in this case.

As this will passed in to the express `app.METHOD` call stack it will rely on `next` to either continue or propagate errors. The call signature for the `callback` expects `void` or `Promise<void>` for the response.

Reply:
done